.github/CODEOWNERS

```diff
@@ -0,0 +1 @@
+*.xtend @adpi2 @ckipp01 @jastice
```

Comment:
So the idea here is that whenever someone updates the actual protocol (anything touching the xtend files) we'll all be automatically assigned to review. We can either review ourselves or add someone else from out teams to do this.

Reply:
That's a nice feature.

Reply:
shall we add smithy files in here right now to not forget it later?

Reply:
There might be some small edits as we merge smithy in and tweak things. So I'd recommend getting it in first, making sure it's working as expected, and then we can rid the codebase of any and all `xtend` references and replace them with smithy.